cdap-data-fabric/src/main/java/co/cask/cdap/spi/data/table/field/Range.java

```diff
@@ -106,6 +105,15 @@ public static Range singleton(Collection<Field<?>> singleton) {
     return new Range(singleton, Bound.INCLUSIVE, singleton, Bound.INCLUSIVE);
   }
 
+  /**
+   * Create a range that matches all the elements.
+   *
+   * @return a range object
+   */
+  public static Range all() {
```

Comment:
this should be a static final constant.

Reply:
I propose we use the method version to keep it consistent with other methods like from, to, etc.